ShareMenu/ShareViewController.swift

```diff
@@ -117,7 +117,7 @@ extension ShareViewController {
         
         let dispatchGroup = DispatchGroup()
         
-        for attatchment in inputItem.attachments ?? [] {
+        for attatchment in inputItem.attachments ?? [] { // swiftlint:disable:this optional_default_value
```

Comment:
<!--
  0 failure: 
  1 warning:  'optional_default...
  
  
  DangerID: danger-id-Danger;
  File: ShareMenu/ShareViewController.swift;
  Line: 120;
-->

- :warning: 'optional_default_value' is not a valid SwiftLint rule. Please remove it from the disable command. (`superfluous_disable_command`)